### ISSUE TITLE
How to fine-tune on another dataset？

### ISSUE THREAD

**Issue by stoneshi-1999** [Rem105-210/DiffFashion#6]
![image](https://github.com/Rem105-210/DiffFashion/assets/70521393/1dd65de8-31df-4682-91d9-934a7f81f01d)
![image](https://github.com/Rem105-210/DiffFashion/assets/70521393/3811af51-ff7e-46a6-ada8-87e11dfaa1ea)

Combining the above two flowcharts, my understanding is that the method is only tuned in the inference phase?

That is, during denoising at the inference stage, only DINO-VIT (VIT_LOSS) is used to compute structure loss and appearance loss?

So the OceanBag Dataset is actually only used for evaluation, not for training?

How should I fine-tune it on other datasets if I expect the method to be more suitable for other datasets?

Looking forward to your reply, thank you so much~

**Comment by Rem105-210** [comment]
Yes, our method only operates on images at inference stage, like neural style transfer methods. DINO-ViT works only in the inference phase and the OceanBag dataset is only used for testing. If you want to apply this method on other datasets, since both Diffusion and ViT are pre-trained on ImageNet, the inference dataset cannot have too complex a background. And due to the randomness of Diffusion, it is necessary to select appropriate negative labels and hyperparameters, and may run a few times to get a good mask. We have tested it on other clothing types and the code will be updated in the future.
